app/src/main/java/ca/marshallasch/veil/FragmentLogin.java

```diff
@@ -71,12 +71,18 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
             db.close();
 
             // check that a user was found
-            if (user == null) {
+
+            //TODO: (DEBUGGING CODE) [START] RESTORE AFTER FINISH TESTING ===============================================
```

Comment:
What on earth is this, you cant just turn off a block of code, if you want it to remember your login then add in a block to remember the username and password?

Reply:
Shoot I forgot to comment it back in.

Reply:
resolved.